build.gradle

```diff
@@ -24,10 +24,15 @@ apply plugin: 'kotlin'
 targetCompatibility = '1.6'
 sourceCompatibility = '1.6'
 
+sourceSets {
```

Comment:
But kotlin jvm plugin automatically adds sourceSets `src/main/kotlin` and `src/main/test`, it's not necessary to configure them explicitly.